docs/operate/reference/viam-server/manage-viam-server.md

```diff
@@ -156,6 +156,40 @@ brew services restart viam-server
 
 ## Update `viam-server`
 
+The steps to update `viam-server` differ depending on your installation method.
+You can check this by seeing if `viam-agent` is running.
+Run the following command:
+
+```sh {class="command-line" data-prompt="$" data-output="2"}
+ps aux | grep viam-agent
+root      566431  0.5  0.2 1247148 20336 ?       Ssl  11:24   0:00 /opt/viam/bin/viam-agent --config /etc/viam.json
+```
+
+If you see a process running viam-agent, then you used `viam-agent` to install `viam-server`.
+If not follow the steps for the standalone version.
```

Comment:
```suggestion
If the output includes a process named `viam-agent`, you used `viam-agent` to install `viam-server`.
Follow the steps to use `viam-agent` to update `viam-server`.

If the output does not include a process named `viam-agent`, you did not use `viam-agent`.
Follow the manual steps to update the standalone version.
```

Suggestion: Use parallel construction for the 'if' statements beneath the code block and clearer conditionals to make it easier for the reader to determine their next step.